lib/suggestion-list.js

```diff
@@ -245,31 +253,49 @@ export default class SuggestionList {
       }
     } else {
       this.destroyOverlay()
-      this.activeEditor = editor
-      this.displayBufferPosition = bufferPosition
-      const marker = this.suggestionMarker = editor.markBufferRange([bufferPosition, bufferPosition])
-      this.overlayDecoration = editor.decorateMarker(marker, {type: 'overlay', item: this, position: 'tail'})
-      this.addBindings(editor)
+      if (editor) {
+        this.activeEditor = editor
+        this.displayBufferPosition = bufferPosition
+        const marker = this.suggestionMarker = editor.markBufferRange([bufferPosition, bufferPosition])
+        this.overlayDecoration = editor.decorateMarker(marker, {type: 'overlay', item: this.suggestionListElement, position: 'tail', class: 'autocomplete-plus'})
+        const editorElement = atom.views.getView(this.activeEditor)
+        if (editorElement && editorElement.classList) {
+          editorElement.classList.add('autocomplete-active')
+        }
+
+        process.nextTick(() => { this.suggestionListElement.didAttach() })
```

Comment:
@joefitzgerald: what do you think about using `process.nextTick` here? Removing custom elements also prevents us from detecting when the element has been attached. In this case, we will simply signal `SuggestionListElement` that it has been attached on the next tick of the event loop.